common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -91,6 +91,16 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
+    "ab-acquisitions-epic-liveblog-design-test",
+    "This tests some different designs of the liveblog epic",
+    owners = Seq(Owner.withGithub("joelochlann")),
```

Comment:
it's fine - this was a join effort and I've probably got more investment in the results since I did the liveblog epic originally